std/file.d

```diff
@@ -2333,14 +2335,27 @@ void mkdirRecurse(in char[] pathname)
     }
 }
 
-@system unittest
+/// ditto
+void mkdirRecurse(R)(auto ref R pathname)
+if (isConvertibleToString!R)
+{
+    return mkdirRecurse!(StringTypeOf!R)(pathname);
+}
```

Comment:
`mkdir` accepts `auto ref`, so I guessed that it was probably an oversight that mkdirRecurse doesn't?